pkg/volume/iscsi/iscsi_util.go

```diff
@@ -34,7 +35,21 @@ import (
 	"k8s.io/kubernetes/pkg/volume"
 	volumeutil "k8s.io/kubernetes/pkg/volume/util"
 	"k8s.io/kubernetes/pkg/volume/util/volumepathhandler"
-	"strconv"
+)
+
+const (
+	// Minimum number of paths required for a multipath iSCSI volume to continue with mounting the volume.
+	// Any lesser number will be reported as error and such volume won't be mounted.
```

Comment:
@jsafrane is this source comment true , specifically `such volume wont be mounted`?

Reply:
Thanks for spotting this, fixed.